plugins/polio/js/src/domains/Campaigns/Scope/Scopes/MapScope.tsx

```diff
@@ -86,7 +68,7 @@ export const MapScope: FunctionComponent<Props> = ({
                 );
                 return {
                     ...selectedPathOptions,
-                    color: vaccine?.color || theme.palette.primary.main,
+                    color: vaccine?.color || theme.palette.secondary.main,
```

Comment:
I think it would be cleaner to change how we define `vaccine` lines 66-67 with something like

```typescript
    const color = (availableVaccines.find(
                    v => v.value === scope.vaccine,
                ) ?? otherVaccine).color; // where otherVaccine is defined in an exported const
    return { ...selectedPathOptions, color }
```
Or something more legible in the same vein.

again in the spirit of decoupling map colors from theme (or at least theme's secondary color)